tests/test_apiclient.py

```diff
@@ -29,15 +29,18 @@ def test_constructor_with_no_token(self, token):
     @pytest.mark.parametrize('error', get_error_test_cases(
         ['unauthorized', 'job-not-found', 'invalid-job-state']))
     @pytest.mark.parametrize('method', ["POST", "GET", "DELETE"])
-    def test_make_http_request(self, error, method, mock_client, make_mock_response):
+    def test_make_http_request(self, error, method, mock_session, make_mock_response):
         status = error.get('status')
         URL = RevAiAPIClient.base_url
         response = make_mock_response(url=URL, status=status, json_data=error)
-        mock_client.session.request.return_value = response
+        mock_session.request.return_value = response
+        client = RevAiAPIClient("token")
```

Comment:
I think there's some inconsistency that could be cleaned up with the usage of TOKEN, token, and "token"

Reply:
I missed this, I should be using TOKEN everywhere